README.md

```diff
@@ -2,11 +2,17 @@
 
 TensorSets are a [third party resource](http://kubernetes.io/docs/user-guide/thirdpartyresources/) to manage [Tensorflow](https://github.com/tensorflow) training clusters running in [Kubernetes](https://github.com/tensorflow).
 
-Note, this is a duct-tape POC. Using this in production will result in multiple RGEs.
+## What's new
```

Comment:
Elson: We are including a "What's new" and a "Known issues" section in each TAP ATK repo README.md. They will typically be skimpy in content for this first release, but will likely grow for subsequent releases. New features and fixed issues will appear in "What's new" going forward. "Known issues" is self-explanatory. Dan